### PR TITLE
fix: allow users to set the location for their cache

### LINKED ISSUE #118: Allow for someone to specify a different cache directory for AuditJS
This I do not THINK we need to implement but it may be that we do. I could potentially see a use case of setting it to `/tmp` or some other directory in systems where the user does not have a lot of access (mostly for CI/CD potentially)?

Would love to hear others feedback on this!

### PR DESCRIPTION
**Pull request by allenhsieh**
This PR allows users to set the location of their cache by setting it in config.

This resolves #118

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 auditjs.json
 # ci config for local ci build
 /.circleci/local-config.yml
+
+.nyc_output
```
Comment (DarthHater) [inline .gitignore]: TRIVIAL: newline

File: src/Config/AppConfig.ts [hunk 2]
```diff
@@ -66,19 +68,25 @@ export class AppConfig {
       
       return config.saveFile(iqConfig);
     } else {
+      let cacheLocation = join(homedir(), '.ossindex', 'auditjs');
```
Comment (DarthHater) [inline src/Config/AppConfig.ts]: Nice, I knew I forgot something!

File: .gitignore (hunk 1 above)
Comment (allenhsieh) [reply .gitignore]: Lol, in: https://github.com/sonatype-nexus-community/auditjs/pull/146/commits/6d3d7fbb3b2c2066a870c6ec2d92b0f5b09770d7

**Pull request merged**